### PR TITLE
Other foss

### PR DESCRIPTION
**Pull request by kirti3192**

### CODE REVIEW

File: creation/models.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ class FossCategory(models.Model):
     category = models.ManyToManyField(FossSuperCategory)
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
+    show_on_homepage = models.BooleanField(max_length=2,default=1 )
```
Comment (theskumar) [inline creation/models.py]: BooleanField doesn't have `max_length` and default should be `True` instead of `1`.

File: spoken/forms.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ class KeywordSearchForm(forms.Form):
 
 class TutorialSearchForm(forms.Form):
     try:
-        foss_list = TutorialResource.objects.filter(Q(status=1) | Q(status=2), language__name='English').values('tutorial_detail__foss__foss').annotate(
+        foss_list = TutorialResource.objects.filter(Q(status=1) | Q(status=2), language__name='English', tutorial_detail__foss__show_on_homepage = 1).values('tutorial_detail__foss__foss').annotate(
```
Comment (theskumar) [inline spoken/forms.py]: Use `True` instead of `1` here.

File: spoken/forms.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ class TutorialSearchForm(forms.Form):
             widget=forms.Select(),
             required=False,
         )
-        lang_list = TutorialResource.objects.filter(Q(status=1) | Q(status=2)).values('language__name').annotate(
+        lang_list = TutorialResource.objects.filter(Q(status=1) | Q(status=2), tutorial_detail__foss__show_on_homepage = 1).values('language__name').annotate(
```
Comment (theskumar) [inline spoken/forms.py]: Use True instead of 1 here.

File: spoken/forms.py [hunk 4]
```diff
@@ -37,6 +37,32 @@ class TutorialSearchForm(forms.Form):
...
+
+        for foss_row in foss_list:
+            choices.append((str(foss_row[0]), str(foss_row[0]) + ' (' + str(foss_row[1]) + ')'))
+        search_otherfoss = forms.ChoiceField(
+            choices=choices,
+            widget=forms.Select(),
+            required=False,
+        )
+        lang_list = TutorialResource.objects.filter(Q(status=1) | Q(status=2), tutorial_detail__foss__show_on_homepage = 0).values('language__name').annotate(
+            Count('id')).order_by('language').values_list('language__name', 'id__count').distinct()
```
Comment (theskumar) [inline spoken/forms.py]: Use `False` instead of `0`.

**Comment by kirti3192** [comment]
@theskumar : I have done the requested changes. Plz review these

**Pull request merged**